core/pom.xml

```diff
@@ -11,6 +11,11 @@
   <properties>
     <guava.version>20.0</guava.version>
     <gwt.version>2.8.0</gwt.version>
+    <junit.version>4.12</junit.version>
+    <jsr305.version>3.0.1</jsr305.version>
+    <auto-value.version>1.4.1</auto-value.version>
+    <compile-testing.version>0.10</compile-testing.version>
```

Comment:
@cgruber @netdpb @cpovirk 

This appears to be problematic, as `0.10` is built for Java 8. Travis is barking on the java7 builds that need this.

Only [two tests need this](https://github.com/google/truth/blob/cb03a71134ffb9941a82940b72f17fbfec7b9825/core/src/test/java/com/google/common/truth/ComparableSubjectTest.java#L248-L305), and it's really testing an API guarantee. I think we could disable the test on Java 7 if we feel that the test is really necessary.